config/Makefile.mingw

```diff
@@ -168,7 +170,7 @@ SYSTEM=mingw
 NATIVECC=$(BYTECC)
 
 ### Additional compile-time options for $(NATIVECC).
-NATIVECCCOMPOPTS=-DCAML_NAME_SPACE -Wall -Wno-unused
+NATIVECCCOMPOPTS=$(BYTECCCOMPOPTS)
```

Comment:
This doesn't strike me as a good idea - if a user chooses to override `BYTECCCOMPOPTS` (e.g. either by editing `config/Makefile` or by specifying `make -f Makefile.nt BYTECCCOMPOPTS=... world.opt`) then it's strange that that would imply setting native code options too?